CHANGES.md

```diff
@@ -16,24 +16,25 @@ The release versions are PyPi releases.
 #### Fixes
   * fake `DirEntry` now implements `os.PathLike` in Python >= 3.6
     (see [#483](../../issues/483))
-  * fixed incorrect argument name for `os.makedirs` 
+  * fixed incorrect argument name for `os.makedirs`
     (see [#481](../../issues/481))
   * avoid pytest warning under Python 2.7 (see [#466](../../issues/466))
+  * add __next__ to FakeFileWrapper
```

Comment:
Can you please add a reference to the issue like in the other entries?

Reply:
Addressed the point